test/test-remote/looker/src/logs/dummystream.ts

```diff
@@ -48,8 +48,8 @@ import { TimeseriesBase, LabelSet } from "../series";
 // Note: maybe expose raw labels later on again.
 export interface LogSeriesOpts {
   // think: n log entries per stream/series fragment
-  n_samples_per_series_fragment: number;
-  n_chars_per_message: number;
+  n_entries_per_stream_fragment: number;
```

Comment:
(renamed vars here and elsewhere to line up with the equivalent args.ts names - easier backtracking when there's a validation error)